### PR TITLE
feat: add update 82

### PR DESCRIPTION
**Pull request by stringhandler**

### CODE REVIEW

File: _updates/2022-07-08-update-82.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+layout: update
+tag: Developer Update
+date: 2022-07-08
+author: stringhandler
+thumbnail: update-background.jpg
+title: This week in Tari
+class: subpage
+---
+
+Tari's DAN Layer is progressing nicely with a lot of pull requests (PR's) related to checkpoints and committees getting merged this week.
```
Comment (delta1) [inline _updates/2022-07-08-update-82.md]:
```suggestion
Tari's DAN Layer is progressing nicely with a lot of pull requests (PRs) related to checkpoints and committees getting merged this week. 
```

File: _updates/2022-07-08-update-82.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+date: 2022-07-08
+author: stringhandler
+thumbnail: update-background.jpg
+title: This week in Tari
+class: subpage
+---
+
+Tari's DAN Layer is progressing nicely with a lot of pull requests (PR's) related to checkpoints and committees getting merged this week. 
+
+One exciting PR merged this week was **[#4260](https://github.com/tari-project/tari/issues/4260) Add range proof batch verification to validators**. More than 80 percent of CPU time during block sync and verification is taken up by range proof verification. Using a clever trick of using the same cryptographic generators, multiple rangeproofs can be verified at the same time, dramatically cut down the time taken in verification.
```
Comment (delta1) [inline _updates/2022-07-08-update-82.md]:
```suggestion
One exciting PR merged this week was **[#4260](https://github.com/tari-project/tari/issues/4260) Add range proof batch verification to validators**. More than 80 percent of CPU time during block sync and verification is taken up by range proof verification. Using a clever trick of using the same cryptographic generators, multiple range proofs can be verified at the same time, dramatically cutting down the time taken in verification.
```

File: _updates/2022-07-08-update-82.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Here's a list of other PR's recently merged:
```
Comment (delta1) [inline _updates/2022-07-08-update-82.md]:
```suggestion
Here's a list of other PRs recently merged:
```

**Review by delta1 — APPROVED** [review]
Maybe add a link to the committees RFC?

File: _updates/2022-07-08-update-82.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+author: stringhandler
+thumbnail: update-background.jpg
+title: This week in Tari
+class: subpage
+---
+
+Tari's DAN Layer is progressing nicely with a lot of pull requests (PRs) related to checkpoints and committees getting merged this week. 
+
+One exciting PR merged this week was **[#4260](https://github.com/tari-project/tari/issues/4260) Add range proof batch verification to validators**. More than 80 percent of CPU time during block sync and verification is taken up by range proof verification. Using a clever trick of using the same cryptographic generators, multiple range proofs can be verified at the same time, dramatically cutting down the time taken in verification.
+
```
Comment (CjS77) [inline _updates/2022-07-08-update-82.md:15]: Talk about leaving us hanging! Where are the benchmarks showing the speed-up?

**Review by CjS77 — COMMENTED** [review]
One comment.

**Review by CjS77 — APPROVED** [review]
LGTM!

**Pull request merged**